src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -2256,10 +2269,56 @@ public void seekToBeginningAndReceive(TestContext context) {
 
         seekAsync.await();
 
+        // Consumer instance not found
```

Comment:
This added more lines in the test and the limit was exceeded. Please use `@SuppressWarnings("checkstyle:MethodLength")` before method name to suppress checkstyle warning.

Reply:
Fixed